### PR TITLE
Update Tween docs with note for deg_to_rad() to make things more clear

### PR DESCRIPTION
**Pull request by ghost**
Updated Tween documentation with a note/tidbit about deg_to_rad() when doing rotations.

Fixes confusion when using Tween for 3d rotation

### CODE REVIEW

File: doc/classes/Tween.xml [hunk 1]
```diff
@@ -69,6 +69,15 @@
 		In the example above, all children of a node are moved one after another to position (0, 0).
 		You should avoid using more than one [Tween] per object's property. If two or more tweens animate one property at the same time, the last one created will take priority and assign the final value. If you want to interrupt and restart an animation, consider assigning the [Tween] to a variable:
 		[codeblocks]
+		[codeblocks]
```
Comment (AThousandShips) [inline doc/classes/Tween.xml:72]:
```suggestion
```

File: doc/classes/Tween.xml [hunk 2]
```diff
@@ -69,6 +69,15 @@
...
+		[gdscript]
+		tween.tween_property(self, "rotation:x", deg_to_rad(-90), 0.5)
+		[/gdscript]
+		[csharp]
+		tween.TweenProperty(this, "rotation:x", Mathf.DegToRad(-90f), 0.5f)
+		[/csharp]
+		[/codeblocks]
+		When doing a rotation in 3D you must convert degrees to radians for expected results. See the example above
```
Comment (AThousandShips) [inline doc/classes/Tween.xml:80]:
```suggestion
		When doing a rotation in 3D you must convert degrees to radians for expected results. See the example above.
		[codeblocks]
```
This should be moved below the example here, as the line above it refers to it, this makes the example confusing:
> consider assigning the [Tween] to a variable:

**Review by AThousandShips — CHANGES_REQUESTED** [review]

**Comment by akien-mga** [comment]
I think that's not really the place to document this. It's not specific to Tween that to modify `rotation` it's in radians, so you need to use radians, or convert your degrees to radians. You also have `rotation_degrees` exposed as a helper if you prefer to use degrees in code.

**Comment by AThousandShips** [comment]
There's dedicated pages for these things, it's also mentioned on `Node3D`, so it's mentioned in areas people would look to understand these things:
> Unless otherwise specified, all methods that have angle parameters must have angles specified as radians. To convert degrees to radians, use @GlobalScope.deg_to_rad.

Further the mention of 3D is misleading as it's true for 2D as well

**Comment by Mickeon** [comment]
Barring the checks failing, I do not think that this PR is necessary.

Tweens are very broad. They work with a lot of different properties and are not specialized for rotation.
If this note were to be included, other similar quirks should also be included for consistency. However, all of them would bloat this documentation, and are better explained elsewhere, such as in the properties' own descriptions.

**Pull request closed without merging**